packages/package_info_plus/package_info_plus/lib/package_info_plus.dart

```diff
@@ -9,7 +9,7 @@ import 'package:package_info_plus_platform_interface/package_info_platform_inter
 
 export 'src/package_info_plus_linux.dart';
 export 'src/package_info_plus_windows.dart'
-    if (dart.library.html) 'src/package_info_plus_web.dart';
```

Comment:
Can you explain this change?

Reply:
according to this document you need to use js_interop for conditional import to support WASM,
am I wrong?
https://dart.dev/interop/js-interop/package-web